changelog.md

```diff
@@ -1,3 +1,63 @@
+# 4.2.0 / 2022-04-29
+
+### New Features
+
+For users:
+- The user interface is now available in Arabic
+- Handle right-to-left languages on interface
+- Feature improvements: Notifications, Kanban, Agenda
+- UI/UX improvements (responsive, buttons, redirections)
+- External links to pictures now have a preview in comments
+
+For admins:
+- Tracimcli command lines have been added to validate custom properties schema and mail configuration validation
+- Add support for anonymous LDAP and anonymous SMTP authentication has been added
+
+### Fixed Issues
+
+- Backend: [#3929](https://github.com/tracim/tracim/issues/3929),
+[#4536](https://github.com/tracim/tracim/issues/4536),
+[#4706](https://github.com/tracim/tracim/issues/4706),
+[#5334](https://github.com/tracim/tracim/issues/5334),
+[#5378](https://github.com/tracim/tracim/issues/5378),
+[#5495](https://github.com/tracim/tracim/issues/5495),
+[#5539](https://github.com/tracim/tracim/issues/5539),
+[#5558](https://github.com/tracim/tracim/issues/5558)
+[#5559](https://github.com/tracim/tracim/issues/5559),
+- Frontend: [#4813](https://github.com/tracim/tracim/issues/4813),
+[#5313](https://github.com/tracim/tracim/issues/5313),
+[#5333](https://github.com/tracim/tracim/issues/5333),
+[#5471](https://github.com/tracim/tracim/issues/5471),
+[#5494](https://github.com/tracim/tracim/issues/5494),
+[#5506](https://github.com/tracim/tracim/issues/5506),
+[#5507](https://github.com/tracim/tracim/issues/5507),
+[#5513](https://github.com/tracim/tracim/issues/5513),
+[#5535](https://github.com/tracim/tracim/issues/5535),
+[#5558](https://github.com/tracim/tracim/issues/5558),
+[#5575](https://github.com/tracim/tracim/issues/5575),
+[#5593](https://github.com/tracim/tracim/issues/5593)
+- Documentation: [#4578](https://github.com/tracim/tracim/issues/4578),
+[#4862](https://github.com/tracim/tracim/issues/4862),
+[#5360](https://github.com/tracim/tracim/issues/5360),
+[#5573](https://github.com/tracim/tracim/issues/5573)
+
+### Breaking/Important change
+
+- :warning: Radicale has been updated to 3.*. You need to adjust some settings before running this version. For more information see: [#5479](https://github.com/tracim/tracim/issues/5479)
+- :warning: Advanced Search (Elasticsearch) has been improved. If you use it, You need to you will need to rebuild the index. For more information see: [#5479](https://github.com/tracim/tracim/issues/5479)
+- Translations are now handled via Weblate. If you want to contribute, check out this [link](https://github.com/tracim/tracim#translation)
+- Language switch button is now displayed in the account settings
+- Caldav and Webdav links are displayed in the space settings
+- New interface for advanced text edition (TinyMCE has been updated to version 5.10)
+- Kanban is activated by default in docker,  see [#5607](https://github.com/tracim/tracim/issues/5607)
+- Pyramid has been updated to 2.*, see [#5047](https://github.com/tracim/tracim/issues/5047)
```

Comment:
```suggestion
- To avoid right issues with `tracimcli` command used in the docker image, the process to properly run `tracimcli` inside official docker image has been clarified:[see here](https://github.com/tracim/tracim/blob/develop/tools_docker/README.md#tracimcli-inside-docker)
- Pyramid has been updated to 2.*, see [#5047](https://github.com/tracim/tracim/issues/5047)
```